src/core_modules/capture-core/components/D2Form/field/buildField.module.css

```diff
@@ -1,5 +1,5 @@
 .booleanLabel {
-    padding-top: 6px;
+    padding-top: 13px;
```

Comment:
![image](https://user-images.githubusercontent.com/4181674/98117043-7d92a180-1ea9-11eb-887e-ebd5ff8c982a.png)
`booleanLabel` is responsible for what we see in the image. I also tried to align them